packages/@react-aria/table/src/useTableColumnHeader.ts

```diff
@@ -40,30 +42,39 @@ interface ColumnHeaderAria {
  */
 export function useTableColumnHeader<T>(props: ColumnHeaderProps, state: TableState<T>, ref: RefObject<HTMLElement>): ColumnHeaderAria {
   let {node} = props;
+  let allowsSorting = node.props.allowsSorting;
   let {gridCellProps} = useGridCell(props, state, ref);
 
   let isSelectionCellDisabled = node.props.isSelectionCell && state.selectionManager.selectionMode === 'single';
   let {pressProps} = usePress({
-    isDisabled: !node.props.allowsSorting || isSelectionCellDisabled,
+    isDisabled: !allowsSorting || isSelectionCellDisabled,
     onPress() {
       state.sort(node.key);
     }
   });
 
   // Needed to pick up the focusable context, enabling things like Tooltips for example
   let {focusableProps} = useFocusable({}, ref);
-  let ariaSort: HTMLAttributes<HTMLElement>['aria-sort'] = null;
-  if (node.props.allowsSorting) {
-    ariaSort = state.sortDescriptor?.column === node.key ? state.sortDescriptor.direction : 'none';
```

Comment:
I still think we want `aria-sort` on column header.

Reply:
Sounds good, gonna tweak it a bit from what I said in standup:
Going to add `aria-sort` back in for all devices and will only add the extra sort order direction to the `aria-describedby` messaging for Android TalkBack since `aria-sort` doesn't work there at all